cmd/cluster/sre_operators/describeCmd.go

```diff
@@ -0,0 +1,331 @@
+package sre_operators
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/openshift/osdctl/pkg/printer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/xanzy/go-gitlab"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/cmd/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+type sreOperatorsDescribeOptions struct {
+	genericclioptions.IOStreams
+	kubeCli client.Client
+}
+
+type sreOperatorDetails struct {
+	Name      string
+	Current   string
+	Expected  string
+	CsvStatus string
+	Channel   string
+
+	CsvHealthPhase   string
+	CsvHealthMessage string
+
+	SubscriptionMessage string
+
+	OperartorGroupHealth string
+
+	DeploymentHealth string
+	PodHealth        string
+}
+
+const (
+	sreOperatorsDescribeExample = `
+		# Describe SRE operators
+		$ osdctl cluster sre-operators describe <operator-name>
+	`
+	sreOperatorsLongDescription = `
+  Helps obtain various health information about a specified SRE operator within a cluster,
+  including CSV, Subscription, OperatorGroup, Deployment, and Pod health statuses.
+
+  A GitLab access token is required to fetch the latest version of the operator,
+  and can be set in the '.config/osdctl' file as 'gitlab_access: <TOKEN>'.
```

Comment:
Since this is a new config added to osdctl, can we also add this to the setup command to ensure it is setup properly?

Reply:
Hi Pratik thank you very much for the suggestion, added.